### PR TITLE
Brandon createpresenter7

### PR DESCRIPTION
**Pull request by branflake2267**
Content slot selection and some model configuration

### CODE REVIEW

**Comment by branflake2267** [comment]
screen shot 2013-10-11 at 2 34 07 pm

File: src/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                return true;
+            }
+        });
+
+        String[] listData = new String[slots.size()];
+        slots.toArray(listData);
+        contentSlotsList.setListData(listData);
+    }
+
+    public String getContentSlot() {
```
Comment (christiangoudreau) [inline src/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java]: method order

**Comment by christiangoudreau** [comment]
LGTM

File: src/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import com.intellij.util.Query;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+import java.util.ArrayList;
+
+public class ContentSlotDialog extends DialogWrapper {
+
```
Comment (meriouma) [inline src/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java]: Do you have an empty line at the beginning of all classes?

**Comment by meriouma** [comment]
LGTM

File: src/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java (hunk 2 above)
Comment (branflake2267) [inline src/com/arcbees/plugin/idea/dialogs/ContentSlotDialog.java]: no nl should be removed. :)

**Pull request merged**